src/components/BookList.jsx

```diff
@@ -1,25 +1,52 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import BookItem from './BookItem';
+import Book from './Book';
+import AddBook from './AddBook';
 
-const BookList = ({ booksProps }) => (
-  <ul>
-    {booksProps.map((book) => (
-      <BookItem key={book.id} itemProp={book} />
-    ))}
-  </ul>
-);
+function Booklist() {
+  const books = [
+    {
+      id: 1,
+      title: 'Book 1',
+      category: 'Action',
+      author: ' Blessing Michael',
+      completed: '65%',
+      currentChapter: 'Chapter 3',
+    },
+    {
+      id: 2,
+      title: 'Book 2',
+      category: 'Fiction',
+      author: 'Martin Luther',
+      completed: '80%',
+      currentChapter: 'Chapter 1',
+    },
+    {
+      id: 3,
+      title: 'Book 3',
+      category: 'Economy',
+      author: 'Quin Din',
+      completed: '99%',
+      currentChapter: 'Chapter 10',
+    },
+  ];
```

Comment:
- [ ] Is there any need for these initial books considering you have initial books in the bookSlice and that's the one that works? I think you should remove these initial books.